src/mds/StrayManager.cc

```diff
@@ -722,6 +723,7 @@ void StrayManager::migrate_stray(CDentry *dn, mds_rank_t to)
   ceph_tid_t tid = mds->issue_tid();
 
   auto req = make_message<MClientRequest>(CEPH_MDS_OP_RENAME);
+  req->set_reintegrated_op();
```

Comment:
s/set_reintegrated_op/set_reintegrate_op

Reply:
Okay, I forgot to amend the changes. Thanks @dparmar18